pkg/packages.builds

```diff
@@ -3,7 +3,8 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <ItemGroup>
-    <Project Include="$(MSBuildThisFileDirectory)\projects\*\*.builds" />
+  	<ProjectExclusions Include="$(MSBuildThisFileDirectory)\projects\*\Microsoft.NETCore.UniversalWindowsPlatform.builds" Condition="'$(OSEnvironment)' != 'Windows_NT'" />
```

Comment:
This seems like the wrong coupling to me.

Shouldn't the `Microsoft.NETCore.UniversalWindowsPlatform` project do the skipping itself?  It would make more sense to have this "don't build me on non-windows" logic be in that project instead of being up in the higher directories.